util/src/main/proto/Debug.proto

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto2";
+
+package org.datacommons.proto;
+
+//
+// A log of import processing with details on any warnings, errors, etc.
+//
+message Log {
+    message Location {
+        // Name of the file. Could be a full path or not. Typically, this is
+        // an MCF file or CSV.
+        optional string file = 1;
+        // Line number in file, starting from 1.  Matches the row number in
+        // CSV (since we do not handle newlines in values).
+        optional int64 line_number = 2;
+        // Column number in line, starting from 1. Can be empty.
+        optional int64 column_number = 3;
+        // Column name set when the input is a CSV. Can be empty.
+        optional string column_name = 4;
+    }
+
+    // Severity level of the message.
+    enum Level {
+        LEVEL_UNSPECIFIED = 0;
+        LEVEL_INFO = 1;
+        LEVEL_WARNING = 2;
+        LEVEL_ERROR = 3;
+        LEVEL_FATAL = 4;
+    }
+
+    message CounterSet {
+      map<string, int64> counters = 1;
+    }
+
+    // One log entry. This could be a sample of messages for a particular
+    // counter.
+    message Entry {
+        optional Level level = 1;
+        optional Location location = 2;
+        // This must be user understandable.
+        optional string message = 3;
+        // A counter in CounterSet.
```

Comment:
nit: A counter key for CounterSet? (and likewise, field name could be counter_key since counter alone is ambiguous)

Reply:
Done.